### PR TITLE
Fix Bug 1320098 - Add Fennec Nightly to Android channel page

### PR DESCRIPTION
**Pull request by kyoshino**
## Description

I've split the Fennec Nightly addition from #4382 which has contained mutiple bug fixes. 😓 

## Bugzilla link

[Bug 1320098](https://bugzilla.mozilla.org/show_bug.cgi?id=1320098)

## Testing

The [demo server](https://bedrock-demo-channel-android-nightly.us-west.moz.works/en-US/firefox/channel/android/) will be live soon. Make sure the new Nightly section is displayed, and the download link works properly.

## Checklist
- [ ] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/firefox/firefox_details.py [hunk 1]
```diff
@@ -327,12 +328,12 @@ class FirefoxAndroid(_ProductDetails):
...
-    aurora_url_base = ('https://archive.mozilla.org/pub/mobile/nightly/'
-                       'latest-mozilla-aurora-android')
-    aurora_urls = {
-        'api-9': aurora_url_base + '-api-9/fennec-%s.multi.android-arm.apk',
-        'api-15': aurora_url_base + '-api-15/fennec-%s.multi.android-arm.apk',
-        'x86': aurora_url_base + '-x86/fennec-%s.multi.android-i386.apk',
+    archive_url_base = ('https://archive.mozilla.org/pub/mobile/nightly/'
+                        'latest-mozilla-%s-android')
+    archive_urls = {
+        'api-9': archive_url_base + '-api-9/fennec-%s.multi.android-arm.apk',
```
Comment (pascalchevrel) [inline bedrock/firefox/firefox_details.py]: api-9 is Android 2.3.x and I don't think we still support it, looking at our FTP:
https://archive.mozilla.org/pub/mobile/releases/47.0/
https://archive.mozilla.org/pub/mobile/releases/48.0/

It seems we only support api-15 now (Android 4.4+)

Comment (pascalchevrel) [inline bedrock/firefox/firefox_details.py]: yes confirmed in our 48 release notes:
https://www.mozilla.org/en-US/firefox/android/48.0/releasenotes/
>Support for Android 2.3 (Gingerbread) has ended. Farewell!

Comment (kyoshino) [inline bedrock/firefox/firefox_details.py]: Yes, it's actually not used at this moment. I'll refactor the code in [Bug 1293711](https://bugzilla.mozilla.org/show_bug.cgi?id=1293711)

**Comment by kyoshino** [comment]
Looks like my branch name was wrong, there should be 2 underscores. Will send a new PR. Thanks @pascalchevrel for pointing this out!

**Pull request closed without merging**